server/selinux/server/pulp-celery.te

```diff
@@ -55,6 +55,7 @@ allow celery_t pulp_tmp_t:file manage_file_perms;
 files_tmp_filetrans(celery_t, pulp_tmp_t, file)
 
 allow celery_t pulp_var_run_t:file manage_file_perms;
+allow celery_t pulp_var_run_t:dir { write remove_name search add_name };
```

Comment:
Consider replacing this with `manage_dir_perms`. I think that would be a good change. It's a little less fine grained, but I think it captures the intent of what you are going for.